### ISSUE TITLE
IFrame content doesn't scale in Chrome

### ISSUE THREAD

**Issue by goeddea** [hakimel/reveal.js#462]
When using IFrames on slides, in Chrome the IFrame container expands/contracts when the browser window is resized. The IFrame content retains its size.

The scaling of IFrame content works fine in FireFox and IE.

(all of the above on Windows 7)

When commenting out the use of zoom.js in the 'layout' function, Chrome scales correctly. 
(comment out lines 678-82 + 690 in release 2.4.0)

**Comment by edemaine** [comment]
`iframe`s still don't scale correctly when `zoom` is active on Chrome for Windows.  I found the following workaround, which copies the `zoom` value (if present) to a `transform` on iframes.

```js
Reveal.on('resize', () => {
  slides = document.querySelector('.reveal .slides');
  zoom = slides.style.zoom;
  document.querySelectorAll('iframe').forEach((iframe) => {
    if(zoom && zoom > 1) {
      iframe.style.transform = "scale(#{zoom})";
      iframe.style.transformOrigin = 'top left';
    } else {
      iframe.style.transform = '';
    }
  })
});
```

This fixes the issue for me on Chrome on Windows, and as far as I can tell via BrowserStack, works on MacOS too. Can anyone confirm?

This wouldn't be compatible if someone manually specified a `transform` style on an `iframe`, but otherwise seems like a nice addition. Perhaps it would be worth adding to the documentation, at least?

**Comment by edemaine** [comment]
Ugh, this has weird behavior too. It's good sometimes, but definitely not always.

But I see from https://github.com/hakimel/reveal.js/commit/e281b3234e7991283ce4dcca705dd9a6a9ebe5d2 that you're planning to remove the `zoom` reliance altogether. Awesome! I'm very much looking forward to a release with that. ❤️

**Comment by hakimel** [comment]
As @edemaine mentioned, this has been fixed in master e281b3234e7991283ce4dcca705dd9a6a9ebe5d2 and will be released with the next version.

Using zoom to scaling has caused a lot of headaches over the years :S I really wanted to hold on to it for the best possible sharpness but I can't justify the amount of quirks that comes with it.
